File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}
        for num in nums:
            freq[num] = freq.get(num, 0) + 1
        
        buckets = [[] for _ in range(len(nums) + 1)]
        for num, f in freq.items():
            buckets[f].append(num)
        
        res = []
        for i in range(len(buckets) - 1, -1, -1):
            for num in buckets[i]:
                res.append(num)
                if len(res) == k:
                    return res

File: Data Structures & Algorithms/concatenation-of-array/submission-1.go
func getConcatenation(nums []int) []int {
    res := []int{}
    for i:=0;i<len(nums);i++{
        res = append(res, nums[i])
    }

    for i:=0;i<len(nums);i++{
        res = append(res, nums[i])
    }
    return res
}
